openlane/steps/step.py

```diff
@@ -721,7 +722,7 @@ def filename_with_counter():
         dumpable_config = copy_recursive(self.config, translator=visitor)
         dumpable_config["meta"] = {
             "openlane_version": __version__,
-            "step": self.__class__.id,
+            "step": self.__class__.id.rstrip("-0123456789"),
```

Comment:
If we merge #255 we could just use `_original_id`

Reply:
_original_id is only returned by method `with_id`. How should I utilize that here ?

Reply:
i guess in `__init_subclass__` we can check if the class hasattr `_original_id` and if not, we just set it to the id?

Reply:
The problem is self.__class__.id varies when in a flow there a step is repeated. I am not sure where this happens but perhaps at that point `_original_id` could be set to id without numbers

Reply:
I… implemented something